src/vconsole/vconsole-setup.c

```diff
@@ -53,6 +54,41 @@ static bool is_vconsole(int fd) {
         return ioctl(fd, TIOCLINUX, data) >= 0;
 }
 
+static int toggle_utf8(int fd, bool utf8) {
```

Comment:
Actually it would be nicer to merge this patch with the following patch which removes the old functions. It's harder to see what actually changed when the addition and removal is separated.